src/main/java/uk/gov/hmcts/reform/notificationservice/service/NotificationService.java

```diff
@@ -41,7 +41,8 @@ public void processPendingNotifications() {
 
     @Transactional(readOnly = true)
     public List<Notification> findByFileNameAndService(String fileName, String service) {
-        log.info("Getting notifications for file {}, service {}", fileName, service);
+        final String fileNameForLogPattern = fileName.replaceAll("[\n|\r|\t]", "_");
+        log.info("Getting notifications for file {}, service {}", fileNameForLogPattern, service);
 
         return notificationRepository.find(fileName, service);
```

Comment:
we can pass secured string down the pipeline - will be safe in any scenario then

Reply:
I agree - entry to the service is the right place for things like that. We could simply validate file name.

Reply:
@doncem @lgonczar Have done

Reply:
Passing it down to the repository is fine, but what I had in mind what that we should check the file in `NotificationController.getNotifications()`, where this value enters the system. Then it will be safe to use anywhere.

Reply:
Should we validate file_name and return BAD REQUEST in case of failure?